### DISCUSSION TITLE
Flutter: Different between ListviewBuilder and Listview

### DISCUSSION THREAD

**Discussion by anushamanp** [community/community#21307] [Mobile] [answered]
Dear Team,

I really get confused about ListviewBuilder and Listview,
can anyone help me

**Comment by surajadkhari** [comment, marked as answer]
So, in short, ListView() construct all the child, that could possibly be displayed in the list view instead of just those children that are actually visible., and with ListView.builder() the builder is called only for those children that are actually visible.

![flutter](https://user-images.githubusercontent.com/48079501/180939633-b6f3551f-2ac5-4d9a-8f54-73532ef67ab7.jpg)

![flutter2](https://user-images.githubusercontent.com/48079501/180939641-c424242e-334b-436d-89d4-3bf849e52eec.jpg)

**Reply by anushamanp** [reply]
Thank you for sharing
